core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1584,6 +1584,10 @@ public enum Property {
       "The interval at which to check for external compaction final state markers in the metadata table.",
       "2.1.0"),
   @Experimental
+  COMPACTION_COORDINATOR_FINALIZER_QUEUE_SIZE(
```

Comment:
I think this might need to be added to `Property.fixedProperties`, it's not dynamic.

Reply:
Update that in 0e874d1